interface/forms/gad7/report.php

```diff
@@ -12,19 +13,32 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
+/* these are included in gad7.inc.php
 require_once(dirname(__FILE__) . '/../../globals.php');
 require_once($GLOBALS["srcdir"] . "/api.inc");
+*/
+require_once("gad7.inc.php");
 
+$gad7_total = 0;
+$pdf_as_string = '';
+$data;
+$exp = '';
+$file_path = $GLOBALS['fileroot'] . '/tmp/'; /* where to store the pdfs */
```

Comment:
can't figure out where above $file_path is being used

Reply:
and address this question too please